.github/workflows/test-report.yml

```diff
@@ -32,4 +32,6 @@ jobs:
         run: ./gradlew generateTestReport
 
       - name: Update tracking issue
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
minor nitpick to mirror  the names here, but if its a difficult change that's fine.